intercept.go

```diff
@@ -130,9 +130,16 @@ func messageCommand(session *discordgo.Session, e *discordgo.Message, guild *dis
 			return
 		}
 
-		inMS := timestamp.Sub(first).Nanoseconds() / time.Millisecond.Nanoseconds()
-		text += "Incoming: `" + strconv.FormatInt(inMS, 10) + "ms`"
-		text += "Calculating outgoing.."
+		inMS := first.Sub(timestamp).Nanoseconds() / time.Millisecond.Nanoseconds()
+
+		// Discord 'bug' makes us receive the message before the timestamp, sometimes.
+		text := ""
+		if inMS < 0 {
+			text += "Incoming: `" + strconv.FormatInt(inMS, 10) + "ms`"
+		  text += "Calculating outgoing.."
```

Comment:
Still, that text is never cleared later. Also, I don't think you meant for it to be in the if check